CMakeLists.txt

```diff
@@ -122,7 +122,22 @@ endif()
 if( NOT CMAKE_BUILD_TYPE )
   set( CMAKE_BUILD_TYPE Release )
 endif()
-set( FALLBACK_VERSION "Uncrustify-0.64_f" )
+
+# get a fallback version string from the package.json file to guard against
+# failure of the make_version.py script
+set(FALLBACK_VERSION "")
+file(READ "package.json" content)
+string(REPLACE "\n" ";" lines "${content}")
+foreach(line ${lines})
+    string(STRIP "${line}" line)
+    if("${line}" MATCHES "^\"version\" *: *\"([0-9.]+)\"")
```

Comment:
The line is stripped at 132 so there will be no white spaces. 
Unfortunately CMake can only deal with simple regex, so no character classes.

The last sentence I don't understand completely. I am enforcing (fairly loosely) that only digits and dots can be used for the version. Inside `make_version.py` I am more strictly but as this here is used as fallback it should be alright.